Station.h

```diff
@@ -136,6 +136,7 @@ class Station
      *  \param freq0 %Station beam former reference frequency (Hz).
      *  \param station0 %Station beam former reference direction (ITRF, m).
      *  \param tile0 Tile beam former reference direction (ITRF, m).
+     *  \param rotate Boolean deciding if paralactic rotation should be applied.
```

Comment:
To understand this issue in future edits, we should probably explain the full issue in the description of this method.